contracts/modules/cover/Cover.sol

```diff
@@ -316,28 +357,79 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
     }
   }
 
-  function requestAllocation(
+  function requestAllocationsForNewCover(
```

Comment:
pls keep names consistent between the 2 functions - something along this lines:
`requestAllocationsNewCover`
`requestAllocationsEditCover`